.github/workflows/release_branch_issue.yml

```diff
@@ -72,3 +75,55 @@ jobs:
           OLD_BODY=$(echo $(gh issue view $TICKET_NUM --json body | jq -r '.body'))
           NEW_BODY=$(echo "$OLD_BODY" | sed -e "s#\^\^\^Testrail-url\^\^\^#$RUN_URL#g")
           gh issue edit $TICKET_NUM -b $NEW_BODY
+  release_coordination:
+    name: release coordination
+    if: ${{ success() }}
+    needs: [release_ticket]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Map GitHub usernames to Slack IDs
+        id: map_slack_ids
+        run: |
+          
+          declare -A GITHUB_TO_SLACK_MAP
+          GITHUB_TO_SLACK_MAP["DJUltraTom"]="U02BBL9L0CU"
+          GITHUB_TO_SLACK_MAP["kellylein"]="UJHA49K6X"
+          GITHUB_TO_SLACK_MAP["dumathane"]="U02RC1BRZBP"
+          GITHUB_TO_SLACK_MAP["timwright12"]="U01DBDAJZ18"
+          GITHUB_TO_SLACK_MAP["ala_yna"]="UQC180926"
+          GITHUB_TO_SLACK_MAP["ajsarkar28"]="U05BNQLJ9UZ"
+          GITHUB_TO_SLACK_MAP["rachelhanster"]="U047Q1LKDFY"
+          GITHUB_TO_SLACK_MAP["DonMcCaugheyUSDS"]="U06PCD1R0EL"
+          GITHUB_TO_SLACK_MAP["IsraelleHub"]="U06N7M2QSUA"
+          GITHUB_TO_SLACK_MAP["rtwel"]="UEY4D750B"
+
+          # Example: Convert mapping to environment variables
+          for key in "${!GITHUB_TO_SLACK_MAP[@]}"; do
+            echo "${key}=${GITHUB_TO_SLACK_MAP[$key]}" >> $GITHUB_ENV
+          done
+
+      - name: Notify Slack for release coordination
+        env:
+          SLACK_API_TOKEN: ${{ secrets.SLACK_API_TOKEN }}
+          SLACK_DJUltraTom: $DJUltraTom
+          SLACK_kellylein: $kellylein
+          SLACK_dumathane: $dumathane
+          SLACK_timwright12: $timwright12
+          SLACK_alayna: $ala_yna
+          SLACK_ajsarkar28: $ajsarkar28
+          SLACK_rachelhanster: $rachelhanster
+          SLACK_DonMcCaugheyUSDS: $DonMcCaugheyUSDS
+          SLACK_rtwel: $rtwel
+        run: |
+          SLACK_MESSAGE=$(cat <<EOF
```

Comment:
Is is possible to thread this message like in the example from the ticket? https://dsva.slack.com/archives/C018V2JCWRJ/p1722428026986259

It would be nice to keep this clean and make sure people use the active thread

Reply:
@timwright12 I updated the workflow. Please review whenever you get a chance

Reply:
@narin @timwright12 Please review

Reply:
Nesting looks good, but we'll those dates filled out too